README.md

```diff
@@ -122,8 +122,13 @@ Also, we also provide Optuna docker images on [DockerHub](https://hub.docker.com
 
 ## Contribution
 
-Any contributions to Optuna are welcome! When you send a pull request, please follow the
-[contribution guide](./CONTRIBUTING.md).
+Any contributions to Optuna are more than welcome!
+
+If you are new to Optuna, please check [good first issues](https://github.com/optuna/optuna/labels/good%20first%20issue). These are simple and well-defined so that they can be good starting points for you to get familiar with the contribution workflow and other developers.
```

Comment:
```suggestion
If you are new to Optuna, please check [good first issues](https://github.com/optuna/optuna/labels/good%20first%20issue). They are simple and well-defined so that they can be good starting points for you to get familiar with the contribution workflow and other developers.
```